docs/event-bus/03-04-knative-eventing-mesh.md

```diff
@@ -41,3 +41,76 @@ To reach the new Eventing Mesh, use an HTTP request with the `/events` path.
 For example, if you have used `gateway.example.cx/v1/events` so far, use `gateway.example.cx/events` to make sure you work with the new Eventing Mesh. 
 
 >**NOTE:** The HTTP source adapter only accepts Events compliant with the [CloudEvents 1.0 specification](https://github.com/cloudevents/spec/blob/v1.0/spec.md).
+
+## Other Channel options
+
+By default Kyma comes with Natss but Knative eventing allows to exchange the default channel implementation. It even allows to have multiple channel implementations simultaneously.
```

Comment:
Thanks for the link :)

Reply:
`implement multiple channels simultaneously.` => I think it should be `use`, because in this documentation we only care about using channels, not implementing them. I would take your suggestion but with the word change. What do you think?